sdc/datatypes/hpat_pandas_series_rolling_functions.py

```diff
@@ -1154,6 +1154,8 @@ def hpat_pandas_series_rolling_var(self, ddof=1):
     """
     Limitations
     -----------
+    This function may reveal slower performance than Pandas* on user system. Users should exercise a tradeoff
```

Comment:
I think it should be converted to list.

Reply:
And in other places where Limitations consist of several statements they should be organized in list.